Current dev doesn't compile with Ocaml 5.1.0 under Linux

I get this error:
```
File "src/syntax/grammar.ml", lines 47-49, characters 8-14:
47 | ........match%parser s with
48 | 		| [ (sep2,_); [%let l = psep_trailing sep f] ] when sep2 = sep -> v :: l
49 | 		| [ ] -> [v]
Error: function expected
make: *** [Makefile:81: haxe] Error 1
```
This new ppx_parser seems to be installed. Anything else I need to do?

--- a/haxe.opam
+++ b/haxe.opam
@@ -27,14 +27,11 @@ depends: [
   "extlib" {>= "1.7.8"}
   "sha"
   "camlp-streams"
-  "ppx_parser"
+  "ppx_parser" {>= "0.2.0"}
   "conf-libpcre2-8"
   "conf-zlib"
   "conf-neko"
   "luv" {>= "0.5.13"}
   "ipaddr"
   "terminal_size"
-]
-pin-depends: [
-  ["ppx_parser.dev" "git+https://github.com/NielsMommen/ppx_parser"]
 ]

